app/src/main/java/org/fossasia/pslab/activity/OscilloscopeActivity.java

```diff
@@ -524,56 +534,48 @@ public void onClick(View v) {
             case R.id.button_channel_parameters_os:
                 replaceFragment(R.id.layout_dock_os2, channelParametersFragment, "ChannelParametersFragment");
                 clearTextBackgroundColor();
-                howToUseDialog(getString(R.string.channel_parameters), getString(R.string.channel_param_desc), R.drawable.mic_schematic, getString(R.string.channel_param_mic));
                 channelParametersTextView.setBackgroundColor(getResources().getColor(R.color.colorPrimaryDark));
                 break;
 
             case R.id.tv_channel_parameters_os:
                 replaceFragment(R.id.layout_dock_os2, channelParametersFragment, "ChannelParametersFragment");
                 clearTextBackgroundColor();
-                howToUseDialog(getString(R.string.channel_parameters), getString(R.string.channel_param_desc), R.drawable.mic_schematic, getString(R.string.channel_param_mic));
                 channelParametersTextView.setBackgroundColor(getResources().getColor(R.color.colorPrimaryDark));
                 break;
 
             case R.id.button_timebase_os:
                 replaceFragment(R.id.layout_dock_os2, timebaseTriggerFragment, "TimebaseTiggerFragment");
                 clearTextBackgroundColor();
-                howToUseDialog(getString(R.string.timebase), getString(R.string.timebase_desc), R.drawable.timebase_view, null);
                 timebaseTiggerTextView.setBackgroundColor(getResources().getColor(R.color.colorPrimaryDark));
                 break;
 
             case R.id.tv_timebase_tigger_os:
                 replaceFragment(R.id.layout_dock_os2, timebaseTriggerFragment, "TimebaseTiggerFragment");
                 clearTextBackgroundColor();
-                howToUseDialog(getString(R.string.timebase), getString(R.string.timebase_desc), R.drawable.timebase_view, null);
```

Comment:
@Vikum94 are these guides gone? I can't figure out how they are shown to user.

Reply:
@CloudyPadmal All are in the same bottom sheet as one whole guide for the instrument